domain/actionSpace/utils.ts

```diff
@@ -26,6 +27,12 @@ export function giveBirthToDwarf(actionSpace: ActionSpace, player: Player): Enti
     ];
 }
 
+function checkPlayerCanHaveMoreDwarf(player: Player) {
```

Comment:
I would have named it with an `assert` given what it returns nothing and throws if something is wrong

Reply:
oki, changed for both!https://github.com/BoredGamesWorkshop/caverna/pull/9/commits/dcbeae1d6c2f4d0b07ebb3e56b2f4873b2a95a88